torch/_inductor/codegen/cpp_micro_gemm.py

```diff
@@ -590,7 +590,11 @@ class CppMicroGemmAMX(CppMicroGemm):
                     C + m * ldc + n,
                     K,
                     lda,
+{%- if not use_cached_dequantized_B %}
                     ldb,
+{%- else %}
+                    block_n,
+{%- endif %}
```

Comment:
Shall we define a new variable for the actual `ldb` so that we don't have to do `{% if -else %}` in several places?

Reply:
Will do, thanks! We might as well do that for `B` as well.